datastore/db/migrations/0023_auto_20240318_1952.py

```diff
@@ -4,45 +4,50 @@
 import django.contrib.postgres.operations
 from django.db import migrations
 
+import os
+
 
 class Migration(migrations.Migration):
 
     dependencies = [
         ("db", "0022_auto_20240227_1648"),
     ]
 
-    operations = [
-        django.contrib.postgres.operations.BtreeGinExtension(),
-        migrations.AddIndex(
-            model_name="grant",
-            index=django.contrib.postgres.indexes.BTreeIndex(
-                fields=["publisher_org_id"], name="db_grant_publish_53bcd4_btree"
+    if os.environ.get("SKIP_SPECIAL_DB_INDEX"):
```

Comment:
I'm so used to static languages, I forget you can do magical things like class-level if statements in Python :magic_wand: